dotCMS/src/main/java/com/dotcms/rest/api/v1/page/ContentletMixIn.java

```diff
@@ -0,0 +1,22 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.portlets.structure.model.Structure;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * Restricts the JSON conversion of specific data in the Contentlet object.
+ *
+ * @author Will Ezell
+ * @version 4.2
+ * @since Oct 9, 2017
+ */
+public abstract class ContentletMixIn {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this "ContentletMixIn" class to an interface [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1694)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "ContentletMixIn" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)